docs/guide/protocols/plainmd.rst

```diff
@@ -1,4 +1,6 @@
 Plain MD Protocol
 =================
 
-issue #743
\ No newline at end of file
+The plain MD protocol enables the user to run an MD simulation of a `ChemicalSystem`, which can contain e.g. a solvated protein-ligand complex, ora ligand and water. 
+The protocol applies a LangevinMiddleIntegrator which uses Langevin dynamics, with the LFMiddle discretization (J. Phys. Chem. A 2019, 123, 28, 6056-6079). 
```

Comment:
I don't think we're using bibtext, so this would be a great one to throw in a reference section for now.